Example/Podfile.lock

```diff
@@ -1,6 +1,6 @@
 PODS:
   - Nimble (7.3.4)
-  - PinwheelSDK (2.2.1)
+  - PinwheelSDK (2.2.0)
```

Comment:
Not a big deal but probably worth rerunning pod install in the Example directory to fix this.